test/periphery/integration/merkle-campaign/lt/MerkleLT.t.sol

```diff
@@ -26,40 +26,35 @@ abstract contract MerkleLT_Integration_Shared_Test is MerkleCampaign_Integration
                                 SHARED TESTS
 //////////////////////////////////////////////////////////////////////////*/
 
-contract Clawback_MerkleLT_Integration_Test is Clawback_Integration_Test, MerkleLT_Integration_Shared_Test {
-    function setUp() public override(Clawback_Integration_Test, MerkleLT_Integration_Shared_Test) {
-        Clawback_Integration_Test.setUp();
+contract Clawback_MerkleLT_Integration_Test is MerkleLT_Integration_Shared_Test, Clawback_Integration_Test {
+    function setUp() public override(MerkleLT_Integration_Shared_Test, MerkleCampaign_Integration_Test) {
```

Comment:
I remember asking `setUp` functions from `Clawback_Integration_Test` once and you told me (I may be wrong about this memory) that it provides a better readability to have explicit overrides. And I agreed so I'd like to make the same suggestion. 

1. bring base the `setUp` in `Clawback_Integration_Test`

then

```solidity
contract Clawback_MerkleLT_Integration_Test is MerkleLT_Integration_Shared_Test, Clawback_Integration_Test {
    function setUp() public override(MerkleLT_Integration_Shared_Test, Clawback_Integration_Test) {
        MerkleLT_Integration_Shared_Test.setUp();
    }
}
```

Otherwise it creates some ambiguity as to why it overrides `MerkleCampaign_Integration_Test` whereas inheriting from `Clawback_Integration_Test`

Reply:
i don’t remember exactly what you’re referring to, sorry. (we didn't have a shared pattern in periphery before your claim PR)

in this PR, we’ve changed how things work, so there’s no point in adding `setUp` functions

Reply:
Wouldn't it be confusing to do it this way?

```solidity
contract Clawback_MerkleLT_Integration_Test is MerkleLT_Integration_Shared_Test, Clawback_Integration_Test {
    function setUp() public override(MerkleLT_Integration_Shared_Test, MerkleCampaign_Integration_Test) {
        MerkleLT_Integration_Shared_Test.setUp();
    }
}
```

vs 

```solidity
contract Clawback_MerkleLT_Integration_Test is MerkleLT_Integration_Shared_Test, Clawback_Integration_Test {
    function setUp() public override(MerkleLT_Integration_Shared_Test, Clawback_Integration_Test) {
        MerkleLT_Integration_Shared_Test.setUp();
    }
}
```

Second one explicitly overrides the inherited contract whereas first one overrides the parent contract. To be able to understand if overrides are accurate, I had to go inside the `Clawback_Integration_Test` to see if all is OK.

Reply:
After thinking about it, I agree with your changes. So consider it resolved from my side.

Reply:
> After thinking about it, I agree with your changes. So consider it resolved from my side.

great, thank you